marklogic-data-hub/src/test/java/com/marklogic/bootstrap/TestAppInstaller.java

```diff
@@ -209,6 +161,78 @@ public void run() {
         }
     }
 
+    /**
+     * When running the tests in DHS, we do not want to install DHF.
+     */
+    private void installDataHubIfNecessary() {
+        HubConfigObjectFactory factory = applicationContext.getBean(HubConfigInterceptor.class).getHubConfigObjectFactory();
+        Properties props = factory.getGradleProperties();
+        if (props.containsKey("hubDhs") && "true".equals(props.getProperty("hubDhs"))) {
```

Comment:
Can we simplify this by using `hubConfig.getIsProvisionedEnvironment()` ?

Reply:
Good catch, did so.